docs/api.rst

```diff
@@ -3262,6 +3262,12 @@ of :class:`enum.Enum`.
         The rule will trigger when combined number of role and user mentions
         is greater than the set limit.
 
+    .. attribute:: member_profile
+
+        The rule will trigger when a user's profile contains a keyword.
```

Comment:
It might be worth clarifying *which* keywords trigger the rule, since this could seem ambiguous otherwise.

Imagine you're somebody reading the documentation - which keywords trigger the rule? Are these keywords explicitly configured? If so, where are they configured?

Reply:
Yeah, the description is based on the description of `.. attribute:: keyword` which is basically based on the same functionality. Also it's just the enum `AutoModTriggerType` which is needed to create a `AutoModTrigger`. There it's better documented inside a table anyway: 
```py
+-----------------------------------------------+------------------------------------------------+
| :attr:`AutoModRuleTriggerType.member_profile` | :attr:`keyword_filter`, :attr:`regex_patterns`,|
|                                               | :attr:`allow_list`                             |
+-----------------------------------------------+------------------------------------------------+
 ```
 So I think it's fine.